tests/metrics/test_classification.py

```diff
@@ -195,3 +203,83 @@ def test_when_no_interactions(self) -> None:
             expected_metric_per_user,
         )
         assert np.isnan(self.metric.calc(RECO, EMPTY_INTERACTIONS))
+
+
+class TestDebiasMetric:
```

Comment:
That's a lot fo `if`s. Seems that making separate classes TestDebiasableClassificationMetric and TestDebiasableSimpleClassificationMetric would have solved the problem